src/verify.v2.integration.test.ts

```diff
@@ -813,11 +1117,427 @@ describe("verify(integration)", () => {
+          },
+          "name": "OpenAttestationDidSignedDocumentStatus",
+          "status": "INVALID",
+          "type": "DOCUMENT_STATUS",
+        },
+      ]
+    `);
+    expect(isValid(results, ["DOCUMENT_STATUS"])).toStrictEqual(false);
+    expect(isValid(results, ["DOCUMENT_INTEGRITY"])).toStrictEqual(true);
+    expect(isValid(results, ["ISSUER_IDENTITY"])).toStrictEqual(false);
+    expect(isValid(results)).toStrictEqual(false);
+  });
+
+  it("should fail with document with missing DID signature", async () => {
+    const results = await verify(documentDidMissingProof, {
+      network: "rinkeby",
+    });
+    expect(results).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          "data": true,
+          "name": "OpenAttestationHash",
+          "status": "VALID",
+          "type": "DOCUMENT_INTEGRITY",
+        },
+        Object {
+          "name": "OpenAttestationSignedProof",
+          "reason": Object {
+            "code": 4,
+            "codeString": "SKIPPED",
+            "message": "Document does not have a proof block",
+          },
+          "status": "SKIPPED",
+          "type": "DOCUMENT_STATUS",
+        },
+        Object {
+          "name": "OpenAttestationEthereumTokenRegistryStatus",
+          "reason": Object {
+            "code": 4,
+            "codeString": "SKIPPED",
+            "message": "Document issuers doesn't have \\"tokenRegistry\\" property or TOKEN_REGISTRY method",
+          },
+          "status": "SKIPPED",
+          "type": "DOCUMENT_STATUS",
+        },
+        Object {
+          "name": "OpenAttestationEthereumDocumentStoreStatus",
+          "reason": Object {
+            "code": 4,
+            "codeString": "SKIPPED",
+            "message": "Document issuers doesn't have \\"documentStore\\" or \\"certificateStore\\" property or DOCUMENT_STORE method",
+          },
+          "status": "SKIPPED",
+          "type": "DOCUMENT_STATUS",
+        },
+        Object {
+          "name": "OpenAttestationDnsTxt",
+          "reason": Object {
+            "code": 2,
+            "codeString": "SKIPPED",
+            "message": "Document issuers doesn't have \\"documentStore\\" / \\"tokenRegistry\\" property or doesn't use DNS-TXT type",
+          },
+          "status": "SKIPPED",
+          "type": "ISSUER_IDENTITY",
+        },
+        Object {
+          "name": "OpenAttestationDnsDid",
+          "reason": Object {
+            "code": 0,
+            "codeString": "SKIPPED",
+            "message": "Document was not issued using DNS-DID",
+          },
+          "status": "SKIPPED",
+          "type": "ISSUER_IDENTITY",
+        },
+        Object {
+          "data": [TypeError: Cannot read property 'find' of undefined],
+          "name": "OpenAttestationDidSignedDidIdentityProof",
+          "reason": Object {
+            "code": 1,
+            "codeString": "UNEXPECTED_ERROR",
+            "message": "Cannot read property 'find' of undefined",
```

Comment:
:eyes:

Reply:
Well.. it's unexpected.
Guess we'll expect it then